Src/IronPython.Modules/socket.cs

```diff
@@ -2862,23 +2862,23 @@ static ssl()
                 return String.Empty;
             }
 
-            [Documentation("read([n]) -> buffer_read\n\n"
-                + "If n is present, reads up to n bytes from the SSL connection. Otherwise, reads to EOF."
-                )]
-            public string read(CodeContext/*!*/ context, [DefaultParameterValue(Int32.MaxValue)] int n) {
+            [Documentation(@"read([len]) -> string
+
+Read up to len bytes from the SSL socket.")]
+            public string read(CodeContext/*!*/ context, [DefaultParameterValue(Int32.MaxValue)] int len) {
                 EnsureSslStream(true);
 
                 try {
                     byte[] buffer = new byte[2048];
-                    MemoryStream result = new MemoryStream(n);
+                    MemoryStream result = new MemoryStream(len);
```

Comment:
Unrelated to the change, but wow, we're really allocating a 2gb array if len is not specified?